packages/universal-crossplane/generated-changes/patch/Chart.yaml.patch

```diff
@@ -2,8 +2,8 @@
 +++ charts/Chart.yaml
 @@ -1,8 +1,12 @@
  apiVersion: v1
--appVersion: 1.2.2-up.1
-+appVersion: 1.2.2001
+-appVersion: 1.2.3-up.1
++appVersion: 1.2.3001
```

Comment:
@turkenh were these version changes intentional? If they were, that's okay to keep. However, if they were made in order to have the upstream chart's version in the rancher package version, it is not necessary and it might be better to keep as `1.2.2`. When you generate a new chart from a package, the upstream chart's version is appended to the package name as following `{chartName}-{chartVersion}{packageVersion}+{upstreamVersion}.tgz`, so in your second commit you will see the following got generated `universal-crossplane-1.2.300100+up1.2.3-up.1.tgz`.

Reply:
@PennyScissors yes this is intentional.

See related issue: https://github.com/upbound/universal-crossplane/issues/119
In summary, without those uxp charts are not visible unless the user explicitly enables "Include Prerelease Versions". We can not just drop `-up.1` part as you suggested since it is part of uxp version. For example, we just released `1.2.3-up.2` which is a different package than `1.2.3-up.2`, and representing both as `1.2.3` in Rancher UI sounds problematic.